### PR TITLE
fix build paddle

### PR DESCRIPTION
**Pull request by guochaorong**

### CODE REVIEW

File: build_paddle.xsh [hunk 1]
```diff
@@ -7,28 +7,29 @@ import config
...
+    "paddle/scripts/paddle_build.sh build",
+    shell=True,
+    cwd=config.paddle_path
+)
+
 
+cd @(config.paddle_path)
+cd build
 pip install --upgrade python/dist/*.whl
+
```
Comment (Superjomn) [inline build_paddle.xsh]: 去空行

**Review by Superjomn — CHANGES_REQUESTED** [review]
跑下 pre-commit run -a

**Comment by guochaorong** [comment]
have done

[root@yq01-gpu-255-120-24-00 continuous_evaluation]# pre-commit run -a
yapf.....................................................................Passed
Check for added large files..............................................Passed
Check for merge conflicts................................................Passed
Check for broken symlinks................................................Passed
Fix End of Files.........................................................Passed
[root@yq01-gpu-255-120-24-00 continuous_evaluation]# git status

**Review by Superjomn — APPROVED** [review]
LGTM

**Pull request merged**